### PR TITLE
(PC-7685) App Native - Profil - Tracking - Redirection dans les réglages systèmes depuis la modale géolocalisation

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-7685

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [x] Added new reusable components to the AppComponents page and communicate to the team on slack

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: src/libs/analytics.ts [hunk 1]
```diff
@@ -174,6 +175,9 @@ const logSearchScrollToPage = (page: number) =>
 const logNoSearchResult = (query: string) =>
   firebaseAnalytics.logEvent(AnalyticsEvent.NO_SEARCH_RESULT, { query })
 
+const logDeeplinkEnableLocation = () =>
```
Comment (blixit) [inline src/libs/analytics.ts:178]: pourquoi "Deeplink" dans le naming ?

Comment (kopax-polyconseil) [reply src/libs/analytics.ts:178]: J'ai gardé les noms demander dans les tickets

File: src/libs/geolocation/components/__tests__/GeolocationActivationModal.test.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      <GeolocationActivationModal
+        hideGeolocPermissionModal={hideGeolocPermissionModal}
+        isGeolocPermissionModalVisible={true}
+        onPressGeolocPermissionModalButton={onPressGeolocPermissionModalButton}
+      />
+    )
+    fireEvent.press(renderAPI.getByText('Activer la géolocalisation'))
+    expect(analytics.logDeeplinkEnableLocation).toBeCalled()
+    expect(onPressGeolocPermissionModalButton).toBeCalled()
+    expect(hideGeolocPermissionModal).not.toBeCalled()
```
Comment (BenStirrup) [inline src/libs/geolocation/components/__tests__/GeolocationActivationModal.test.tsx:21]: Un peu d'aération :)

File: src/libs/analytics.ts (hunk 1 above)
Comment (kopax-polyconseil) [reply src/libs/analytics.ts]: OK

**Review by blixit — APPROVED** [review]

**Pull request merged**